src/enforcement/enforcer.ts

```diff
@@ -123,13 +124,15 @@ export interface IEnforcer {
 export class Enforcer implements IEnforcer {
   public contextStore: ContextStore; // cross-query context (global context)
   private client: AxiosInstance;
+  private opaClient: AxiosInstance;
 
   /**
    * Creates an instance of the Enforcer class.
    * @param config - The configuration object for the Permit SDK.
    * @param logger - The logger instance for logging.
    */
   constructor(private config: IPermitConfig, private logger: Logger) {
+    const opaBaseUrl = `${this.config.opa || 'http://localhost:8181'}/v1/data/permit/`;
```

Comment:
By default, it should use the same URL of the PDP but with a different port.
I would extract the `opaUrl` to a dedicated variable for clarity, use it alone on the `opaAxiosInstance`, and have the path be configured specifically on the check method.